include/crud-terms.php

```diff
@@ -250,6 +250,17 @@ public function unset_tax_query_lang() {
 	 * @return void
 	 */
 	public function delete_term( $term_id ) {
+		$term = get_term( $term_id );
+
+		if ( ! $term instanceof WP_Term ) {
+			return;
+		}
```

Comment:
You don't need this. The taxonomy is accessible as second param of the hook.

Reply:
Done in 6b2153a998a661fedd33ddc022ad59e45cccd6f8